mmv1/third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -604,6 +604,36 @@ func resourceComposerEnvironment() *schema.Resource {
 											},
 										},
 									},
+									<% unless version == "ga" -%>
+									"triggerer": {
+										Type:        schema.TypeList,
+										Optional:    true,
+										Description: `Configuration for resources used by Airflow triggerers.`,
+										MaxItems:    1,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"cpu": {
+													Type:         schema.TypeFloat,
+													Optional:     true,
```

Comment:
are you sure it's optional? According to our yesterday discussion it shall be required (for all fields).

Reply:
Made all triggerer fields required.